etc/docker/tensorflow/1.15.2-py3/Dockerfile

```diff
@@ -15,4 +15,4 @@
 #
 FROM tensorflow/tensorflow:1.15.2-py3
 
-RUN apt install -y curl
+RUN apt update && apt install -y curl
```

Comment:
Could you please add `rm -rf /var/lib/apt/lists/*` after the install to avoid blowing up the image size.

Reply:
done